### PR TITLE
bump mocha version

### PR DESCRIPTION
**Pull request by ngryman**
Following #201.

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -25,6 +25,7 @@
     "istanbul-middleware": "0.2.0",
     "load-script": "0.0.5",
     "lodash": "2.4.1",
+    "mocha": "~2.2.5",
```
Comment (defunctzombie) [inline package.json]: why moved from dev dependencies? Can mocha be used via browserify?

Comment (ngryman) [inline package.json]: I think you could use it with browserify (cf. `mocha/bin/_mocha`).
However, with your current implementation, this is an error of mine.
Will put it back to `devDependencies`.

**Comment by ngryman** [comment]
Moved `mocha` to `devDependencies`.

**Comment by ngryman** [comment]
I have no clue why tests are failing :(

**Comment by vvo** [comment]
@ngryman only because tests are running in saucelabs using credentials not available to PR outside of defunctzombie/

**Comment by ngryman** [comment]
Hey.
Are you planning to merge this PR?
Thanks!

**Comment by vvo** [comment]
@ngryman Can you see if it would be feasible to require mocha programmatically instead of the global copy pasted one?

Also rebase :-)

**Comment by defunctzombie** [comment]
Closing in favor of #227 227

**Pull request closed without merging**